tests/cases/compiler/arrowFunctionParsingGenericInObject.ts

```diff
@@ -0,0 +1,23 @@
+// Apparent parser error when first property of returned object is generic function with default type

```

Comment:
does 'Apparent' mean this *supposed* to be an error? Or is it pasted from a repro?

Reply:
Copy and paste from the issue. I can remove it.